common/src/TopJetIds.cxx

```diff
@@ -6,7 +6,7 @@ using namespace uhh2;
 
 bool CMSTopTag::operator()(const TopJet & topjet, const uhh2::Event &) const {
     auto subjets = topjet.subjets();
-    if(subjets.size() < 3) return false;
+    //if(subjets.size() < 3) return false;
```

Comment:
Hi Doug, I just realized (after accepting the pull request) that you have changed this line. What is the reason for this change? I guess this affects several users of the CMSTopTag routine.